lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -102,7 +102,8 @@ Function Join-Domain {
         [string] $new_hostname,
         [string] $domain_admin_user,
         [string] $domain_admin_password,
-        [string] $domain_ou_path
+        [string] $domain_ou_path,
+        [bool]   $force_replace_host
```

Comment:
Name bikeshedding: how about something like `allow_existing_computer_account`, defaulting to `$false`? `force_replace_X` implies that we're actually replacing the computer account, which isn't true, it just joins the new host with the existing object.